tests/unit/filters/test_phase_to_phase.py

```diff
@@ -0,0 +1,128 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+import numpy as np
+from power_grid_model.enum import WindingType
+from pytest import approx, mark, raises
+
+from power_grid_model_io.filters.phase_to_phase import (
+    get_clock,
+    get_winding_from,
+    get_winding_to,
+    power_wind_speed,
+    reactive_power,
+    relative_no_load_current,
+)
+
+
+@mark.parametrize(
+    ("i_0", "p_0", "s_nom", "u_nom", "expected"),
+    [
+        (float("nan"), float("nan"), float("nan"), float("nan"), float("nan")),
+        (5.0, 1000.0, 100000.0, 400.0, 0.011547005383792516),
+        (5.0, 2000.0, 100000.0, 400.0, 0.02),
+    ],
+)
+def test_relative_no_load_current(i_0: float, p_0: float, s_nom: float, u_nom: float, expected: float):
+    actual = relative_no_load_current(i_0, p_0, s_nom, u_nom)
+    assert actual == approx(expected) or (np.isnan(actual) and np.isnan(expected))
+
+
+def test_relative_no_load_current__exception():
+    with raises(ValueError, match="can't be more than 100% .* 115.47%"):
+        relative_no_load_current(500.0, 1000.0, 100000.0, 400.0)
+
+
+@mark.parametrize(
+    ("p", "cos_phi", "expected"),
+    [
+        (float("nan"), float("nan"), float("nan")),
+        (1000.0, 0.9, 484.3221048378525),
+    ],
+)
+def test_reactive_power(p: float, cos_phi: float, expected: float):
+    actual = reactive_power(p, cos_phi)
+    assert actual == approx(expected) or (np.isnan(actual) and np.isnan(expected))
+
+
+@mark.parametrize(
+    ("wind_speed", "expected"),
+    [
+        (0.0, 0.0),
+        (1.5, 0.0),
+        (3.0, 0.0),  # cut-in
+        (8.5, 125000.0),
```

Comment:
I seem to not get this. Power is supplied in 3 to 14 for default values.
At 8.5, 
power = 1e6 * ((8.5-3)/(14-3))^3 = 324951.171875

Reply:
Hm... on my calculator 1e6 * ((8.5-3)/(14-3))^3 = 1e6 * (5.5/11)^3 = 1e6 * 0.5^3 = 125000.0